packages/project-access/src/index.ts

```diff
@@ -1,5 +1,5 @@
 export { FileName, DirName } from './constants';
-export { getFilePaths } from './file';
+export { getFilePaths, readJSON } from './file';
```

Comment:
Wondering if we should just export everything from here? As its providing all the `memFs` parameters as well.

Reply:
Is there a reason these functions have not been exposed yet @Klaus-Keller ?

Reply:
-> https://github.com/SAP/open-ux-tools/pull/1840#pullrequestreview-2007513845

Reply:
Thank you for explaination, all makes sense.